### DISCUSSION TITLE
Some weird window appear at the top of browser preview ?

### DISCUSSION THREAD

**Discussion by alanmaulanaa** [quarto-dev/quarto-cli#9868] [Q&A]
### Description

So I experienced something that I think could be quite annoying in the future, so when after rendering, and viewing the preview results (either in VSCode or the Browser directly) it appears like either a notification or something above the browser window with "Render" written on it. 

I'm still guessing it only shows up every time I run a new preview command. But this turned out to be quite an annoying thing over time. Is there a solution?
![weird](https://github.com/quarto-dev/quarto-cli/assets/123195469/efaa6aed-cf61-4d7c-8f78-9aa92fb2ba75)

Quarto version : 1.5.40

**Comment by mcanouil** [comment]
No and that's not a bug.
If the page you are previewing gets changed then it needs to be rendered again.
Same goes for pages you access that had no previous render.
It's normal to display that something is happening.

**Reply by alanmaulanaa** [reply]
Is there an option to remove it? Because in the previous version that I used (1.4.554), this never appeared.

**Reply by mcanouil** [reply]
> Is there an option to remove it? Because in the previous version that I used (1.4.554), this never appeared.

I don't believe this to be true as this VSCode behaviour is quite not new.

- https://github.com/quarto-dev/quarto-cli/discussions/6436

FYI, VSCode extension repository is https://github.com/quarto-dev/quarto/issues

**Reply by cscheid** [reply]
@mcanouil There was a windows bug in 1.4 and 1.5 that (silently!) prevented quarto-preview.js from compiling, so I believe windows 1.4 installs until 1.4.555 lack the preview functionality.

**Reply by mcanouil** [reply]
oh, so the bug was not that it appeared but that it disappeared in this particular version.

Thanks @cscheid I did not linked this bug and the "window" in preview.
